### ISSUE TITLE
 pid directory "/run/motioneye" does not exist or is not writable

### ISSUE THREAD

**Issue by xsherlockpl** [motioneye-project/motioneye#2553]
Hi,
I try installing motioneye dev 0.43 on the latest RPI4 running PiOS bullseye that would let it run under python3
Raspberry Pi OS Lite
Release date: April 4th 2022
System: 32-bit
Kernel version: 5.15
Debian version: 11 (bullseye)

It install and but will not start with error  `CRITICAL: pid directory "/run/motioneye" does not exist or is not writable`
a simple sudo mkdir /run/motioneye  fixes that and they I can 
sudo systemctl restart  motioneye

and it is up and running as service.
But that repeats on the reboot.  Is there a way to fix it permanently?

**Comment by JamBalaya56562** [comment]
Thanks for the report! This looks like it's already resolved: the current `motioneye.systemd` unit (`motioneye/extra/motioneye.systemd`) declares:
```
RuntimeDirectory=motioneye
LogsDirectory=motioneye
StateDirectory=motioneye
```
`RuntimeDirectory=` tells systemd to create `/run/motioneye` (owned by the `User=motion` account) before starting the service and clean it up afterwards, so the directory no longer needs to be created manually and persists correctly across reboots. This directive has actually been in the shipped unit file since PR #2353 (merged 2022-03-14), so any current install using `linux_init`/the packaged systemd unit shouldn't hit this. I think this can be closed; if it still reproduces on a fresh install, please mention the exact install method used (systemd vs. manual/Docker) so the remaining case can be tracked down.

**Comment by MichaIng** [comment]
Right, this was actually solved already with the earliest v0.43/Python 3 merge: #2313
I guess the systemd unit was not updated via `motioneye_init` int his case. Closing this.
